generated/nidcpower/nidcpower/_converters.py

```diff
@@ -493,7 +428,7 @@ def test_repeated_capabilies_slice_prefix():
 
 
 def test_repeated_capabilies_from_init():
-    test_result = convert_repeated_capabilities_from_init((slice(0, 1), '2', [4, '5-6'], '7-9', '11:14', '16, 17'), '')
+    test_result = convert_repeated_capabilities_from_init((slice(0, 1), '2', [4, '5-6'], '7-9', '11:14', '16, 17'))
```

Comment:
You had to change this test for a reason. Testing `''` made sense, I think.
Are we ok with this behavior change?

Reply:
I didn't get your question.

Reply:
I misread the code.